src/app/forms/input/input.html

```diff
@@ -0,0 +1,356 @@
+        </div>
+    </div>
+
+    <div class="clr-form-control clr-form-control-disabled" [ngClass]="{row: grid}">
+        <label for="{{layout}}-disabled" class="clr-control-label" [ngClass]="{'col-xs-12': grid, 'col-lg-2': grid && layout !== 'vertical'}">
+            Basic input, disabled
+        </label>
+        <div class="clr-control-container" [ngClass]="{'col-xs-12': grid, 'col-lg-10': grid && layout !== 'vertical'}">
+            <div class="clr-input-wrapper">
+                <input type="text" id="{{layout}}-disabled" placeholder="Enter value here" class="clr-input" value="test value" disabled>
+                <clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>
+            </div>
+            <span class="clr-subtext">Helper Text</span>
+        </div>
+    </div>
+
+    <div class="clr-form-control" [ngClass]="{row: grid}">
+        <label for="{{layout}}-basic-error" class="clr-control-label" [ngClass]="{'col-xs-12': grid, 'col-lg-2': grid && layout !== 'vertical'}">
+            Basic input, error</label>
+        <div class="clr-control-container clr-error" [ngClass]="{'col-xs-12': grid, 'col-lg-10': grid && layout !== 'vertical'}">
+            <div class="clr-input-wrapper">
+                <input type="text" id="{{layout}}-basic-error" placeholder="Enter value here" class="clr-input">
+                <clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>
+            </div>
+            <span class="clr-subtext">Helper Text</span>
+        </div>
+    </div>
+
+    <div class="clr-form-control" [ngClass]="{row: grid}">
+        <label class="clr-control-label" [ngClass]="{'col-xs-12': grid, 'col-lg-2': grid && layout !== 'vertical'}">
+            Basic checkbox</label>
+        <div class="clr-control-container" [ngClass]="{'col-xs-12': grid, 'col-lg-10': grid && layout !== 'vertical'}">
+            <div class="clr-checkbox-wrapper">
+                <input type="checkbox" name="checkbox1" id="{{layout}}-checkbox1" placeholder="Enter value here" class="clr-checkbox"> 
+                <label for="{{layout}}-checkbox1" class="clr-control-label">
+                    option 1
+                </label>
+                <input type="checkbox" name="checkbox2" id="{{layout}}-checkbox2" placeholder="Enter value here" class="clr-checkbox"> 
+                <label for="{{layout}}-checkbox2" class="clr-control-label">
+                    option 2
+                </label>
+                <input type="checkbox" name="checkbox3" id="{{layout}}-checkbox3" placeholder="Enter value here" class="clr-checkbox"> 
+                <label for="{{layout}}-checkbox3" class="clr-control-label">
+                    option 3
+                </label>
+            </div>
+            <!-- IMPORTANT DIFFERENCE IN STRUCTURE! ICON IS NOT PART OF THE INPUT WRAPPER -->
+            <clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>
```

Comment:
Does the icon have to be outside? What would happen here if we used inline checkboxes here? I think we should add a demo for inline checkboxes in all form layouts to test this out.

Reply:
I mean inline checkboxes with validation.

Reply:
saw the demo below